docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   fpm:
-    image: akeneo/fpm:php-7.1
```

Comment:
I think using `akeneo/php:7.2` is enough here. There is no use for FPM.
Plus, could you add what we did on the PIM `docker-compose.yml` file for the composer cache, with variable environments and `docker-compose.override.dist`?